fe/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "tpt-loane",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.10",
+    "@testing-library/react": "^11.2.6",
+    "@testing-library/user-event": "^12.8.3",
+    "@types/jest": "^26.0.22",
+    "@types/node": "^12.20.10",
+    "@types/react": "^17.0.3",
+    "@types/react-dom": "^17.0.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "typescript": "^4.2.4",
+    "web-vitals": "^1.1.1"
```

Comment:
is any of this necessary to run code in production? if not move to dev dependencies. I know it might be weird that there are no dependencies at all but that's how it is.

Reply:
Good point, for some reason they were there with the boilerplate. Moved in [310bef](https://github.com/TPT-Loane/TPT-Loane/pull/1/commits/310befb15c6100c1e42e8fbb4fcb9b45684ef50c)